Purchases/Public/EntitlementInfo.swift

```diff
@@ -134,6 +134,15 @@ import Foundation
      */
     @objc public let ownershipType: PurchaseOwnershipType
 
+    /**
+     * The underlying data for this `EntitlementInfo`.
+     *
+     * - Note: the content and format of this data isn’t documented and is subject to change.
+     *         it's only meant for debugging purposes or for getting access to future data
```

Comment:
capitalization: s/`it's`/`It's`

Reply:
Thanks I meant to have a `,` instead of `.` in the previous line.